test/e2e/clusterctl_upgrade_test.go

```diff
@@ -23,14 +23,68 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 )
 
-var _ = Describe("When testing clusterctl upgrades [clusterctl-Upgrade]", func() {
+var _ = Describe("When testing clusterctl upgrades (v0.3=>current)", func() {
```

Comment:
should this be 
```suggestion
var _ = Describe("When testing clusterctl upgrades (v0.3=>current) [clusterctl-Upgrade]", func() {
```
same for the others

Reply:
if we are not using it in prow, ok to drop it

Reply:
Yup. Won't be used anymore in test-infra after this PR: https://github.com/kubernetes/test-infra/pull/27621 (which is waiting for the current PR)

Double-checked documentation no occurence